datafusion/expr/src/expr_fn.rs

```diff
@@ -119,7 +119,46 @@ pub fn placeholder(id: impl Into<String>) -> Expr {
 /// assert_eq!(p.to_string(), "*")
 /// ```
 pub fn wildcard() -> Expr {
-    Expr::Wildcard { qualifier: None }
+    Expr::Wildcard {
+        qualifier: None,
+        options: WildcardOptions::default(),
+    }
+}
+
+/// Create an '*' [`Expr::Wildcard`] expression with the wildcard options
+pub fn wildcard_with_options(options: WildcardOptions) -> Expr {
+    Expr::Wildcard {
+        qualifier: None,
+        options,
+    }
+}
+
+/// Create an 't.*' [`Expr::Wildcard`] expression that matches all columns from a specific table
+///
+/// # Example
+///
+/// ```rust
+/// use datafusion_common::TableReference;
+/// use datafusion_expr::{qualified_wildcard};
```

Comment:
If you put a `#` in front of lines in an example they are not shown but still compiled

```suggestion
/// # use datafusion_common::TableReference;
/// # use datafusion_expr::{qualified_wildcard};
```